client/src/components/App.test.js

```diff
@@ -3,7 +3,14 @@ import { shallow, mount, render } from 'enzyme';
 
 // Components
 import App from './App';
-import Review from './Review'
+
+/* What tests can I write to help build out App?
+  1. State starts out empty.
+  2. State changes when component mounts - changes to a list of all reviews.
+  3. The App does indeed render.
+  4. 
+
```

Comment:
I really like the questions, but we should remove comments before pushing to master.